cmd/authority-claimer/src/sender.rs

```diff
@@ -190,20 +179,23 @@ impl TransactionSender for DefaultTransactionSender {
     async fn send_rollups_claim_transaction(
         self,
         rollups_claim: RollupsClaim,
-    ) -> Result<Self, Self::Error> {
+        iconsensus: Address,
+    ) -> Result<(H256, Self), Self::Error> {
         let dapp_address = rollups_claim.dapp_address.clone();
 
+        let iconsensus = {
+            let (provider, _mock) = Provider::mocked();
+            let provider = Arc::new(provider);
+            let address: H160 = iconsensus.into_inner().into();
+            IConsensus::new(address, provider)
+        };
+
         let transaction = {
-            let input_range = InputRange {
-                first_index: rollups_claim.first_index as u64,
-                last_index: rollups_claim.last_index as u64,
-            };
-            let call = self
-                .iconsensus
+            let call = iconsensus
                 .submit_claim(
                     H160(dapp_address.inner().to_owned()),
-                    input_range,
-                    rollups_claim.epoch_hash.into_inner(),
+                    U256([rollups_claim.last_block, 0, 0, 0]),
```

Comment:
Just double-checking: should the zeros be on the right?

Reply:
Confirming, they should. Ran a quick test to see the values

```
node-1                    | 2024-08-28T13:52:39.087 INF NUMBER FIRST >>>>>>>>>>> 10 service=authority-claimer
node-1                    | 2024-08-28T13:52:39.087 INF NUMBER LAST >>>>>>>>>>> 62771017353866807638357894232076664161023554444640345128960 service=authority-claimer
```